Dockerfile

```
@@ -1,8 +1,10 @@
-FROM opensuse/tumbleweed AS elixir-build
+FROM opensuse/leap:15.6 AS elixir-build
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
-RUN zypper -n in make gcc git-core elixir elixir-hex erlang-rebar3
+RUN zypper ar https://download.opensuse.org/repositories/devel:sap:trento:builddeps/15.6 buildeps
+RUN zypper -n --gpg-auto-import-keys ref
+RUN zypper -n in make gcc git-core elixir115 elixir-hex erlang26 erlang-rebar3
```

let's keep using version operators instead of explicit package names, it's arguably cleaner and it leaves room for an easier switch to more relaxed requirements.
```suggestion
RUN zypper -n in make gcc git-core elixir==1.15 elixir-hex erlang==26 erlang-rebar3
```